docs/user/firststeps.rst

```diff
@@ -110,40 +133,59 @@ repository:
     # Output similar to:
     # commit ef542b5ec5a44fdbb16afc3de413308a7daff32f
     # Author: John Doe <john.doe@example.com>
-    # Date:   Thu Aug 23 11:58:34 2018 +0000
+    # Date:   Mon Apr 29 11:58:34 2019 +0000
     #
-    #     renku dataset add zhbikes https://data.stadt-zuerich.ch/dataset/verkehrszaehlungen_werte_fussgaenger_velo/resource/d17a0a74-1073-46f0-a26e-46a403c061ec/download/2
-    # 017_verkehrszaehlungen_werte_fussgaenger_velo.csv
+    #     renku dataset add zhbikes https://data.stadt-zuerich.ch/dataset/verkehrszaehlungen_werte_fussgaenger_velo/resource/d17a0a74-1073-46f0-a26e-46a403c061ec/
+    # download/2019_verkehrszaehlungen_werte_fussgaenger_velo.csv
     #
     # commit 38ac3261e8b2964c4608a6ca6d30a4f907dc6930
     # Author: John Doe <john.doe@example.com>
-    # Date:   Thu Aug 23 11:58:30 2018 +0000
+    # Date:   Mon Apr 29 11:56:30 2019 +0000
     #
     #     renku dataset create zhbikes
     #
     # commit 3f74a2dfdf5e27c1dc124f6455931089023253b8
     # Author: John Doe <john.doe@example.com>
-    # Date:   Thu Aug 23 11:55:41 2018 +0000
+    # Date:   Mon Apr 29 11:53:41 2019 +0000
     #
     #     init renku repository
 
+.. code-block:: console
+
     git status
-    # Expected output:
+    # Output similar to:
     # On branch master
     # Your branch is ahead of 'origin/master' by 2 commits.
     #   (use "git push" to publish your local commits)
+    #
     # nothing to commit, working directory clean
 
 Let's push the two fresh commits by running:
 
 .. code-block:: console
 
     git push
+    # Output similar to:
+    # Locking support detected on remote "origin". Consider enabling it with: [...]
+    # Counting objects: 19, done. (1/1), 66 MB | 0 B/s
+    # Delta compression using up to 8 threads.
+    # Compressing objects: 100% (15/15), done.
+    # Writing objects: 100% (19/19), 2.26 KiB | 463.00 KiB/s, done.
+    # Total 19 (delta 3), reused 0 (delta 0)
+    # To https://renkulab.io/gitlab/john.doe/zurich-bikes-tutorial.git
+    #     b55aea9..91b226b  master --> master
+
+The data file can be opened from JupyterLab by going to the **File** tab
+on the top left (1) and traversing the ``data`` folder (2).
```

Comment:
```suggestion
on the top left (1) and clicking the ``data`` folder (2).
```

Reply:
changed slightly since the file is in a sub folder

Reply:
ok